cli/prisma2/package.json

```
@@ -1,6 +1,8 @@
 {
   "version": "2.0.0-alpha.968",
   "name": "@prisma/cli",
+  "description": "CLI for Prisma's open-source database toolkit. It replaces traditional ORMs and makes database access easy with an auto-generated query builder for TypeScript & Node.js.",
+  "keywords": ["prisma2", "prisma", "cli", "database", "postgresql", "mysql", "sqlite", "mariadb", "typescript"],
```

```suggestion
  "keywords": ["prisma2", "prisma", "cli", "database", "postgresql", "mysql", "sqlite", "mariadb", "typescript", "orm", "query-builder"],
```
not sure about these - @2color?